code/game/mob/living/carbon/human/human_life.dm

```diff
@@ -227,8 +227,6 @@
 					mood += 0.12
 				else
 					mood -= 0.4
-		if (!inducedSSD)
-			mood -= 0.02
```

Comment:
why is this in your weapons/armor update

Reply:
forgot to remove while trying to see whats broken with mood system